pkg/controller/kieapp/kieapp_controller.go

```diff
@@ -157,14 +166,14 @@ func (reconciler *Reconciler) Reconcile(request reconcile.Request) (reconcile.Re
 		if err != nil {
 			return reconcile.Result{}, err
 		}
+
 		hasUpdates = hasUpdates || added || updated || removed
 	}
 	if hasUpdates && status.SetProvisioning(instance) {
 		return reconciler.UpdateObj(instance)
 	} else if !hasUpdates {
 		log.Debug("Reconcile loop did not find any resource changes to apply")
 	}
-
```

Comment:
Why is this empty line bad, but two empty lines needed to be added in 160 and 169? :-D

Reply:
Reverting, it was just myself moving code along to make up my mind. Didn't notice